index/scorch/mergeplan/merge_plan.go

```diff
@@ -150,6 +150,7 @@ var DefaultMergePlanOptions = MergePlanOptions{
 var SingleSegmentMergePlanOptions = MergePlanOptions{
 	MaxSegmentsPerTier:   1,
 	MaxSegmentSize:       1 << 30,
+	MaxSegmentFileSize:   1 << 40,
```

Comment:
This won't cause an overflow with int64 would it?

Reply:
nope it doesn't